mdx/guides/abcs-of-contract-fillable-liquidity.mdx

```diff
@@ -1,49 +1,43 @@
-0x protocol's features and networked liquidity can extend and expand the capabilities of your DeFi Protocol. Easily trade tokens atomically on behalf of your users, swap tokens to pay protocol transaction fees, and enable complex liquidation processes by integrating 0x liquidity into your smart contracts. 
```

Comment:
Can we just call this "Introduction to Using 0x Liquidity in your Smart Contracts"